### PR TITLE
docstring added (issue #54)

### PR DESCRIPTION
**Pull request by MihirsinhChauhan**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -13,6 +13,12 @@ nav:
   - Tutorials:
       - Fitting a function: fit_function.ipynb
       - QAOA : QAOA.ipynb
+  - Documentation:
```
Comment (dominikandreasseitz) [inline mkdocs.yml:16]: can we maybe have this under "Overview".

**Comment by dominikandreasseitz** [comment]
thank you! but the issue has been addressed in https://github.com/pasqal-io/PyQ/pull/72. i will definitely have a look at both and see if we can combine stuff.

**Comment by MihirsinhChauhan** [comment]
should need to do any changes and if it is, please give suggestion regarding same

**Comment by dominikandreasseitz** [comment]
hi @MihirsinhChauhan, unfortunately we cant go ahead with this MR since its a duplicate! thank you so much for wanting to contribute though!

**Pull request closed without merging**